.github/workflows/clean_cache.yml

```diff
@@ -0,0 +1,34 @@
+on:
+  pull_request:
+    branches: 
+      - master
+    types:
+      - closed
+
+jobs:
+  clear_cache:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install and setup mc client
+        run: |
+          wget https://dl.min.io/client/mc/release/linux-amd64/mc
+          chmod +x mc
+          ./mc --help
+          ./mc alias set hetzner https://hel1.your-objectstorage.com ${{ secrets.HETZNER_S3_ACCESS_KEY }} ${{ secrets.HETZNER_S3_ACCESS_SECRET }}
+      
+      - name: Remove matching cache objects
+        shell: bash
+        run: |
+          # CAREFUL: only bash, not zsh compatible
```

Comment:
I was despairing here.. I default to zsh and this is the first time I got to know some of the differences to bash processing. how it treats arrays, the output being word-/ or newline-separated etc. quite painful tbh..